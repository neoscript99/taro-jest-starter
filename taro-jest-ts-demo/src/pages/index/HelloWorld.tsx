import Taro, { Component } from '@tarojs/taro'
import { View, Text } from '@tarojs/components'

class HelloWorld extends Component {
  render () {
    return (
      <View><Text>Hello, World</Text></View>
    )
  }
}

export default HelloWorld
