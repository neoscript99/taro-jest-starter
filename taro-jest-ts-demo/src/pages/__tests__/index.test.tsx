import Nerv from 'nervjs'
import { renderToString } from 'nerv-server'
import HelloWorld from '../../../.temp/pages/index/HelloWorld'
import configStore from '../../store'

const store = configStore()

test('with redux', () => {
  const component = renderToString(
    <HelloWorld />
  )
  expect(component)
    .toMatchSnapshot()
})
