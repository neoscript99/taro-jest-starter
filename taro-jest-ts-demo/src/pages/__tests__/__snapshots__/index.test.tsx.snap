// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`with redux 1`] = `"<div class=\\"\\"><span class=\\"taro-text\\">Hello, World</span></div>"`;
